src/locale/locales/fr/messages.po

```diff
@@ -4294,6 +4385,10 @@ msgstr "Code QR a été téléchargé !"
 msgid "QR code saved to your camera roll!"
 msgstr "Code QR enregistré dans votre photothèque !"
 
+#: src/tours/Tooltip.tsx:111
+msgid "Quick tip"
+msgstr "Conseil rapide"
```

Comment:
`Quick tip` is quite a thing by itself; it's not really a `tip` that is `quick`. I prefer using `Petite astuce` that convey the same meaning while being more natural in French.

```suggestion
msgstr "Petite astuce"
```